libraries/AP_Soaring/AP_Soaring.cpp

```diff
@@ -142,16 +142,16 @@ const AP_Param::GroupInfo SoaringController::var_info[] = {
     // 19 reserved for POLAR_LEARN.
 
     // @Param: THML_ARSPD
-    // @DisplayName: Specific setting for airspeed when thermalling.
-    // @Description: If non-zero this airspeed will be used when thermalling.
-    // @Values: 0: Use TRIM_ARSPD_CM, >0: Target airspeed in m/s
+    // @DisplayName: Specific setting for airspeed when in a thermal in SOARING.
```

Comment:
suggest " @DisplayName: Specific setting for airspeed when soaring in THERMAL mode"

Reply:
done